### DISCUSSION TITLE
Codeforces  B.coin 58B

### DISCUSSION THREAD

**Discussion by Eyup02grd** [sonsuzus/sonsuzus#460] [Soru Cevap]
Berland'da bir reform hazırlanıyor. Yeni madeni paralar tanıtılıyor. Uzun ekonomik programlardan sonra en pahalı madeni paranın tam olarak n Berland dolarının ödenmesine karar verildi. Ayrıca, bir madeni paranın değeri, daha ucuz bir madeni paranın bölünebilir olması gerekir. Tüm olası kapasiteler arasında en fazla sayıda yeni madeni para içeren özelliklerin seçildiği belirtilmiştir. Bu özellikleri bulun. Madeni paraların değerlerinin azalan sırasına göre yazdırılması.    

```

print("en yüksek para birimini giriniz ")
n = int(input())

para = []
for i in range(1, n + 1):
    if n % i == 0:
        para.append(i)

para.sort(reverse=True)

print(*para)

```

**Comment by sonsuzus** [comment]
Kod kısmını code etiketi içine almalısın. Ayrıca soruyu iyi çevirdin mi tam anlaşılmıyor? 

Berland'da bir para reformu hazırlanıyor. Yeni madeni paralar tanıtılıyor. Uzun ekonomik hesaplamalardan sonra en pahalı madeni paranın tam olarak n Berland doları değerinde olması gerektiğine karar verildi. Ayrıca rahatlık açısından şu kısıtlama getirilmiştir: her bir madeni paranın değeri, daha ucuz bir madeni paranın değerine bölünebilir olmalıdır. Tüm olası varyantlar arasında en fazla sayıda yeni madeni para içeren varyantın seçileceği bilinmektedir. Bu varyantı bulun. Madeni paraların değerlerinin azalan sırasına göre yazdırın.

Examples
InputCopy
10
OutputCopy
10 5 1
InputCopy
4
OutputCopy
4 2 1
InputCopy
3
OutputCopy
3 1

**Reply by Eyup02grd** [reply]
düzelttim hocam
